packages/tables/src/Columns/Concerns/CanFormatState.php

```diff
@@ -269,4 +291,29 @@ public function isMarkdown(): bool
     {
         return (bool) $this->evaluate($this->isMarkdown);
     }
+
+    public function isDate(): bool
+    {
+        return (bool) $this->isDate;
```

Comment:
No need to cast to bool here, right? The property is boolean already.

Reply:
You're right. I'll fix tomorrow. Bed time. Thanks! 

Hope you're well!

Reply:
Good night! Mark it as ready for review when you've managed to implement the changes. 🙂

Reply:
Done!